lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -233,13 +233,13 @@ defmodule Kernel.SpecialForms do
   Some types have associated modifiers to clear up ambiguity in byte
   representation.
 
-  Modifier             | Relevant Type(s)
-  -------------------- | ----------------
-  `signed`             | `integer`
-  `unsigned` (default) | `integer`
-  `little`             | `integer`, `float`, `utf16`, `utf32`
-  `big` (default)      | `integer`, `float`, `utf16`, `utf32`
-  `native`             | `integer`, `utf16`, `utf32`
+  Modifier               | Relevant Type(s)
```

Comment:
the changes in this table only add italics to default, the rest of the changes is to line everything up in the table.